pkg/cmd/roachtest/tests/kv.go

```diff
@@ -118,7 +118,8 @@ func registerKV(r registry.Registry) {
 			}
 		}
 		if opts.sharedProcessMT {
-			createInMemoryTenant(ctx, t, c, appTenantName, c.Range(1, nodes), true /* secure */)
+			startOpts = option.DefaultStartSharedVirtualClusterOpts(appTenantName)
+			c.StartServiceForVirtualCluster(ctx, t.L(), c.Range(1, nodes), startOpts, install.MakeClusterSettings(), c.Range(1, nodes))
```

Comment:
Sort of proving your point in https://github.com/cockroachdb/cockroach/pull/119679#discussion_r1504730005: the third argument could be _anything_ and it's irrelevant in this case, since we're creating an in-memory tenant. 😓

Reply:
Yeah I wasn't sure to leave it empty or have it be the same as the storage nodes. Both seemed mildly confusing to me 😅